acceptance-tests/src/test/resources/log4j2.xml

```diff
@@ -2,13 +2,30 @@
 <Configuration status="INFO">
   <Properties>
     <Property name="root.log.level">DEBUG</Property>
+    <Property name="dependency.log.level">WARN</Property>
   </Properties>
 
   <Appenders>
     <Console name="Console" target="SYSTEM_OUT">
-      <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss.SSSZZZ} | %t | %-5level | %c{1} | %msg%n" />    </Console>
+      <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss.SSSZZZ} | %t | %-5level | %c{1} | %msg%n" />
+    </Console>
+    <Console name="SubProcessConsole" tareget="SYSTEM_OUT">
+      <PatternLayout pattern="SubProcess | %msg%n" />
+    </Console>
   </Appenders>
   <Loggers>
+    <Logger name="tech.pegasys.web3signer.SubProcessLog" level="INFO" additivity="false">
+      <AppenderRef ref="SubProcessConsole" />
+    </Logger>
+    <Logger name="com.zaxxer.hikari" level="${dependency.log.level}" additivity="false"/>
+    <Logger name="org.flywaydb.core" level="${dependency.log.level}" additivity="false"/>
+    <Logger name="org.apache.http" level="${dependency.log.level}" additivity="false"/>
+    <Logger name="io.restassured.internal" level="${dependency.log.level}" additivity="false"/>
+    <Logger name="io.netty" level="${dependency.log.level}" additivity="false"/>
+    <Logger name="io.swagger.v3.parser.OpenAPIV3Parser" level="${dependency.log.level}" additivity="false"/>
```

Comment:
Do we just want set all swagger logs to warn? i.e. io.swagger as the name. And similarly for com.atlassian.oai?

Reply:
👍

Reply:
I think all of the "dependent" libraries logs should be warn ... unless we want to debug those libraries themselves.

Reply:
yeah, that's fine.